packages/messaging/lib/index.d.ts

```diff
@@ -628,9 +694,9 @@ export namespace FirebaseMessagingTypes {
      *   });
      * ```
      *
-     * @param options Options to override senderId (iOS) and projectId (Android).
+     * @param options Options can be either `GetTokenOptions` or `NativeTokenOptions`
```

Comment:
When I made the original suggestion I had to actually look up the typescript typing syntax and then I still steered you wrong because I misunderstood the typescript type syntax. Many apologies.

This did not seem correct to me though so I've just re-re-read the typescript docs and what we want here is an intersection type (symbol `&`) so we get a composite object with all properties, not just the intersecting properties (which would be empty in this case!)

```suggestion
     * @param options Options composite type with all members of `GetTokenOptions` and `NativeTokenOptions`
```